code/modules/jobs/job_types/tribals.dm

```diff
@@ -55,18 +56,19 @@ Tribal Chief
 	..()
 	if(visualsOnly)
 		return
+	ADD_TRAIT(H, TRAIT_LIFEGIVER, src)
 	ADD_TRAIT(H, TRAIT_BIG_LEAGUES, src)
 
 /datum/outfit/job/tribal/f13chief
 	name = "Chief"
 	jobtype = /datum/job/tribal/f13chief
 	head = 			/obj/item/clothing/head/helmet/f13/wayfarer/chief/green
-	uniform = 		/obj/item/clothing/under/f13/tribe_chief
+	uniform = 		/obj/item/clothing/under/f13/wayfarer
 	belt = 			/obj/item/storage/backpack/spearquiver
 	neck =			/obj/item/clothing/neck/cloak/chiefcloak
 	id = 			/obj/item/card/id/tribetattoo
-	suit =			/obj/item/clothing/suit/armor/f13/tribe_armor
-	suit_store =	/obj/item/twohanded/spear/bonespear/deathclaw
+	suit =			/obj/item/clothing/suit/hooded/cloak/hhunter
+	suit_store =	/obj/item/melee/transforming/cleaving_saw
```

Comment:
Two cleaving saws is quite unlikely to be a popular change. The Hhunter already still has one of their own.

Reply:
Dclaw spear is pretty bad for a leadership role of the faction, currently due to the lack of 2 tile range weaponry, ill look about reverting it once I get around to fixing glaives.